src/test_workflow/integ_test/distribution_deb.py

```diff
@@ -30,6 +30,10 @@ def install(self, bundle_name: str) -> None:
         logging.info("deb installation requires sudo, script will exit if current user does not have sudo access")
         deb_install_cmd = " ".join(
             [
+                'sudo'
+                'env'
+                'OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!'
```

Comment:
This will only set the env variable for the duration of the `sudo env` command execution and will not persist. Recommend to use `export` to set the environment variable.